avocado/utils/network/interfaces.py

```diff
@@ -128,6 +128,65 @@ def add_ipaddr(self, ipaddr, netmask):
         except Exception as ex:
             raise NWException("Failed to add address {}".format(ex))
 
+    @property
+    def vlans(self):
+        list_of_vlan_ids = {}
+        for i in self.host.interfaces:
+            details = i._get_interface_details()
+            if self.name == details.get("link"):
+                list_of_vlan_ids[details.get('linkinfo').get('info_data').get('id')] = details.get("ifname")
+        return list_of_vlan_ids
+
+    def add_vlan_tag(self, vlan_num, vlan_name=None):
+        """Configure 802.1Q VLAN tagging to the interface
+
+        This method will try to add a VLAN tag to this interface and return the
+        VLAN NetworkInterface object. If it fails, the method will raise a
```

Comment:
This method is not returning the VLAN NetwrokInterface anymore. Documentation must be updated.